matchms/filtering/require_precursor_mz_above_0.py

```diff
@@ -0,0 +1,29 @@
+from ..typing import SpectrumType
+from ..typing import Union
+
+
+def require_precursor_mz_above_0(spectrum_in: SpectrumType
+                                 ) -> Union[SpectrumType, None]:
+
+    """Returns None if there is no precursor_mz or if <=0
```

Comment:
```suggestion
def require_precursor_mz(spectrum_in: SpectrumType
                         ) -> Union[SpectrumType, None]:

    """Returns None if spectrum has no valid precursor_mz.
```

Reply:
I think we can leave out the `_above_0` part, but that would also have to be added to the rest of your code (tests...).